src/main/webapp/app/entities/attachment/attachment.service.ts

```diff
@@ -50,6 +50,30 @@ export class AttachmentService {
         return this.http.delete<any>(`${this.resourceUrl}/${id}`, { observe: 'response' });
     }
 
+    downloadAttachment(downloadUrl: string) {
```

Comment:
Good that you extract this into the service!
Would be good if you could add some JSDoc, too.

Reply:
done